From 2826a1164a76d0c7e99d813076071a90e539283f Mon Sep 17 00:00:00 2001
From: TheScarastic <warabhishek@gmail.com>
Date: Fri, 13 Nov 2020 13:43:05 +0100
Subject: [PATCH] copy backuptool files to system

For some reason in android11 it fails to copy to $OUT/install, so copy to system as a temporary workaround till the root cause is discovered

Change-Id: I14f8d38436a955f4fe8f491cfe7be28b58a2984e
Signed-off-by: SamarV-121 <samarvispute121@gmail.com>
---
 config/common.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 657902a..6e5f8ad 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -23,8 +23,8 @@ endif
 
 # Backup Tool
 PRODUCT_COPY_FILES += \
-    vendor/lineage/prebuilt/common/bin/backuptool.sh:install/bin/backuptool.sh \
-    vendor/lineage/prebuilt/common/bin/backuptool.functions:install/bin/backuptool.functions \
+    vendor/lineage/prebuilt/common/bin/backuptool.sh:$(TARGET_COPY_OUT_SYSTEM)/install/bin/backuptool.sh \
+    vendor/lineage/prebuilt/common/bin/backuptool.functions:$(TARGET_COPY_OUT_SYSTEM)/install/bin/backuptool.functions \
     vendor/lineage/prebuilt/common/bin/50-lineage.sh:$(TARGET_COPY_OUT_SYSTEM)/addon.d/50-lineage.sh
 
 ifneq ($(AB_OTA_PARTITIONS),)
-- 
2.28.0

